homeassistant/components/roku/media_player.py

```diff
@@ -76,21 +83,37 @@
     | SUPPORT_BROWSE_MEDIA
 )
 
+
+STREAM_FORMAT_TO_MEDIA_TYPE = {
+    "dash": MEDIA_TYPE_VIDEO,
+    "hls": MEDIA_TYPE_VIDEO,
+    "ism": MEDIA_TYPE_VIDEO,
+    "m4a": MEDIA_TYPE_MUSIC,
+    "m4v": MEDIA_TYPE_VIDEO,
+    "mka": MEDIA_TYPE_MUSIC,
+    "mkv": MEDIA_TYPE_VIDEO,
+    "mks": MEDIA_TYPE_VIDEO,
+    "mp3": MEDIA_TYPE_MUSIC,
+    "mp4": MEDIA_TYPE_VIDEO,
+    "wma": MEDIA_TYPE_MUSIC,
```

Comment:
`wma` support was deprecated by Roku, and removed since firmware v10.5
https://developer.roku.com/en-gb/docs/references/deprecated-apis.md

Reply:
```suggestion
```